CONTRIBUTING.md

```diff
@@ -4,8 +4,11 @@
 
 ## Contributing Time-Series Features
 
-We gratefully accept contributions of new time-series features, be they
-domain-specific or general. Please follow the below guidelines in order that
-your features may be successfully incorporated into the Cesium feature base.
-
-*Coming soon*
+We gratefully accept contributions of new time-series features, be they domain-specific or general. Please follow the guidelines below so that your features are successfully incorporated into the `cesium` feature base.
```

Comment:
How about (2nd sentence).  "To contribute a new feature, follow these steps:" or "You may add a new feature as follows:"